src/main/java/org/opentripplanner/middleware/models/JourneyState.java

```diff
@@ -29,55 +29,63 @@ public JourneyState(MonitoredTrip monitoredTrip) {
     }
 
     /**
-     * The {@link MonitoredTrip} id that this journey state is tracking.
+     * The arrival/departure delay seen in the latest monitored trip check.
      */
-    public String monitoredTripId;
+    public int lastArrivalDelay;
+    public int lastDepartureDelay;
 
     /**
-     * User ID for {@link OtpUser} that owns the {@link MonitoredTrip}.
+     * Timestamp checking the last time a journey was checked.
      */
-    private String userId;
+    public long lastCheckedMillis;
 
     /**
-     * Timestamp checking the last time a journey was checked.
+     * The notifications already sent.
+     * FIXME this is never set, so it has no effect.
      */
-    public long lastChecked;
+    public Set<TripMonitorNotification> lastNotifications = new HashSet<>();
 
     /**
-     * Store the recent plan requests made over the course of monitoring a trip. Note: these should be cleared once the
-     * monitored trip clears for the day (i.e., if the monitored trip occurs at 9am, responses will stack up as we check
-     * the trip. At 9:01am (or perhaps some later time in the day) this should be cleared.).
-     *
-     * FIXME: Should the type be string/responseBody instead?
+     * The last time a notification was sent.
+     * FIXME this is never accessed anywhere and might not be worth persisting.
      */
-    public Response lastResponse;
-
-    public int matchingItineraryIndex;
+    public long lastNotificationTimeMillis;
 
-    public Set<TripMonitorNotification> lastNotifications = new HashSet<>();
-
-    public long lastNotificationTime;
+    /**
+     * The current or upcoming matching itinerary from plan requests made over the course of monitoring a trip.
+     */
+    public Itinerary matchingItinerary;
 
-    public Set<LocalizedAlert> lastSeenAlerts = new HashSet<>();
```

Comment:
Why were lastSeenAlerts removed?

Reply:
This variable wasn't used anymore in the currently implemented logic for checking alerts, so I removed it. See https://github.com/ibi-group/otp-middleware/blob/c9499ccea48990678ed14e52b0761cff80aba104/src/main/java/org/opentripplanner/middleware/tripMonitor/jobs/CheckMonitoredTrip.java#L166-L185